kinesumer.go

```diff
@@ -638,10 +640,19 @@ func (k *Kinesumer) Refresh(streams []string) {
 }
 
 // Errors returns error channel.
+// you must read from the Errors() channel or Kinesumer will ignore error if channel is blocked.
 func (k *Kinesumer) Errors() <-chan error {
 	return k.errors
 }
 
+func (k *Kinesumer) sendErrorOrIgnore(err error) {
+	select {
+	case k.errors <- err:
+	default:
+		// if there are no error listeners, error is ignored.
```

Comment:
In this case, it would be more accurate to say that the error is discarded rather than ignored. So we can rename this method to `sendOrDiscardError`.